providers/github/connection/platform.go

```diff
@@ -19,26 +19,29 @@ const (
 )
 
 var (
-	GithubRepoPlatform = &inventory.Platform{
-		Name:    "github-repo",
-		Title:   "GitHub Repository",
-		Family:  []string{"github"},
-		Kind:    "api",
-		Runtime: "github",
-	}
-	GithubUserPlatform = &inventory.Platform{
-		Name:    "github-user",
-		Title:   "GitHub User",
-		Family:  []string{"github"},
-		Kind:    "api",
-		Runtime: "github",
-	}
-	GithubOrgPlatform = &inventory.Platform{
-		Name:    "github-org",
-		Title:   "GitHub Organization",
-		Family:  []string{"github"},
-		Kind:    "api",
-		Runtime: "github",
+	GithubRepoPlatform = inventory.Platform{
+		Name:                  "github-repo",
+		Title:                 "GitHub Repository",
+		Family:                []string{"github"},
+		Kind:                  "api",
+		Runtime:               "github",
+		TechnologyUrlSegments: []string{"github", "repo"},
+	}
+	GithubUserPlatform = inventory.Platform{
+		Name:                  "github-user",
+		Title:                 "GitHub User",
+		Family:                []string{"github"},
+		Kind:                  "api",
+		Runtime:               "github",
+		TechnologyUrlSegments: []string{"github", "user"},
+	}
+	GithubOrgPlatform = inventory.Platform{
+		Name:                  "github-org",
+		Title:                 "GitHub Organization",
+		Family:                []string{"github"},
+		Kind:                  "api",
+		Runtime:               "github",
+		TechnologyUrlSegments: []string{"github", "org"},
```

Comment:
I'm not sure about this:
Shouldn't "org" match something in the UrlTree? There we have "organization".

Reply:
we can remove them here since we set them in the New* functions